relay-server/src/extractors/request_meta.rs

```diff
@@ -247,7 +247,7 @@ pub struct RequestMeta<D = PartialDsn> {
     ///
     /// NOTE: This is internal-only and not exposed to Envelope headers.
     #[serde(skip)]
-    from_internal_relay: bool,
+    request_trust: Option<RequestTrust>,
```

Comment:
Is there an actual advantage to making this optional? AFAICT `None` has the same meaning as `Some(Untrusted)` in all cases.